src/prover_disk.hpp

```diff
@@ -47,74 +47,52 @@ struct plot_header {
 // The DiskProver, given a correctly formatted plot file, can efficiently generate valid proofs
 // of space, for a given challenge.
 class DiskProver {
+    const size_t nTableBeginPointerCount{11};
```

Comment:
I don't think it's a good idea to add an immutable member like this. This makes `DiskProver` objects not movable.
Did you mean to make this `static`?
Also, I don't think we should use hungarian notation.

Reply:
> This makes `DiskProver` objects not movable.

What do you mean? Maybe not by default but its still movable :man_shrugging: However, i agree, it makes sense to have it `static`. 

> Also, I don't think we should use hungarian notation.

I dropped it in 96f0445 to stay consistent with the codebase, but whats your point here? I'm just curious because i'm used to always use it for primitives or standard container types.

Reply:
I believe there is widespread agreement that encoding type information in variable names is undesirable, as documented in the [cpp core guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rl-name-type).

Reply:
> What do you mean? Maybe not by default but its still movable

Right, I noticed later that you implement the move constructor explicitly, and just don't touch this member.